### PR TITLE
Update to Flux v0.38.0

### PR DESCRIPTION
**Pull request by affo**

### CODE REVIEW

**Comment by affo** [comment]
@e-dard could  you  please re-run  litmus here?

I don't have the permissions to

**Comment by e-dard** [comment]
@affo done.

File: query/stdlib/testing/testing.go [hunk 1]
```diff
@@ -50,6 +50,7 @@ var FluxEndToEndSkipList = map[string]string{
 	"integral":         "unbounded test",
 	"integral_columns": "unbounded test",
 	"map":              "unbounded test",
+	"buckets":          "unbounded test",
```
Comment (jlapacik) [inline query/stdlib/testing/testing.go]: This is a new test that was added that didn't have a `range` call?

Comment (affo) [reply query/stdlib/testing/testing.go]: O sorry, I saw it now, yes

**Pull request merged**